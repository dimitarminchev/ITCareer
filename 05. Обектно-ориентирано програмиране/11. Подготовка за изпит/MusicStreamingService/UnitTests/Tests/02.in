AddUser mitko 28
AddPlaylist mitko Workout
End