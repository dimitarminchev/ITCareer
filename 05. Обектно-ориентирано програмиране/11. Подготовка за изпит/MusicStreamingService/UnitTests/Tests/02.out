Created User mitko!
Created Playlist Workout for User mitko!